### PR TITLE
Fix broken links for shipwright-io/build references

### PR DESCRIPTION
**Pull request by apoorvajagtap**
# Changes

The buildstrategies.md references multiple examples of [build/samples/v1beta1](https://github.com/shipwright-io/build/tree/main/samples/v1beta1) directory. This PR copies the contents from the [build/samples](https://github.com/shipwright-io/build/tree/main/samples/v1beta1) to the current project. This would aid the manual syncs (until we automate) in case of any new changes instead of updating the multiple links each time.

In addition to samples, references to controller.yaml are also updated to refer to the correct link per [#113](https://github.com/shipwright-io/website/pull/113#discussion_r1598899692).

Follow up of #113 

-->

# Submitter Checklist

- [ ] Includes tests if functionality changed/was added
- [ ] Includes docs if changes are user-facing
- [x] [Set a kind label on this PR](https://prow.k8s.io/command-help#kind)
- [x] Release notes block has been filled in, or marked NONE

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by avinal — CHANGES_REQUESTED** [review]
I see there are two types of changes, one for the broken links and the other is newly added YAMLs. 

I would suggest breaking it into at least two commits clearly showing the specific change. Rest looks good to me.

**Comment by apoorvajagtap** [comment]
> I see there are two types of changes, one for the broken links and the other is newly added YAMLs.
> 
> I would suggest breaking it into at least two commits clearly showing the specific change. Rest looks good to me.

@avinal actually the addition of samples/ is done with the same intention of fixing the broken links (for example: https://github.com/shipwright-io/website/blob/main/content/en/docs/build/buildstrategies.md).

**Comment by apoorvajagtap** [comment]
I just realised, that the tests have been failing since my last push. I am currently working on it & shall push the changes soon.

**Comment by adambkaplan** [comment]
Blocked by #127

File: themes/docsy
Comment (adambkaplan) [inline themes/docsy:1]: @apoorvajagtap is there a reason why `docsy` is being updated here?

I also see a lot of other changes that go well beyond the scope of "fixing broken links" - can you explain this further in the body of your commit message?

**Review by adambkaplan — CHANGES_REQUESTED** [review]

File: themes/docsy
Comment (avinal) [reply themes/docsy:1]: +1, docsy changes should be moved to a separate PR.

**Comment by adambkaplan** [comment]
Looks like we incorporated these items in #140 - thanks for the contribution!

**Pull request closed without merging**